extensions/exposition/source/HTTP/headers.ts

```diff
@@ -0,0 +1,53 @@
+import type * as http from 'node:http'
+
+export function logHeaders (headers: http.IncomingMessage['headers']): void {
+  if (headers instanceof Headers)
```

Comment:
I believe this is always `false`. Have you seen headers in the log before pushing?